.devcontainer/devcontainer.json

```diff
@@ -1,39 +1,23 @@
 {
-	"name": "Anaconda (Python 3)",
-	"build": { 
-		"context": "..",
-		"dockerfile": "base.Dockerfile",
-		"args": {
-			"INSTALL_NODE": "true",
-			"NODE_VERSION": "lts/*"
-		}
-	},
+    "name": "Anaconda (Python 3)",
+    "image": "ghcr.io/octodemo-db/github-pyml-development-container:latest",
 
-	// Set *default* container specific settings.json values on container create.
-	"settings": { 
-		"terminal.integrated.shell.linux": "/bin/bash",
-		"python.pythonPath": "/opt/conda/bin/python",
-		"python.linting.enabled": true,
-		"python.linting.pylintEnabled": true,
-		"python.formatting.autopep8Path": "/opt/conda/bin/autopep8",
-		"python.formatting.yapfPath": "/opt/conda/bin/yapf",
-		"python.linting.flake8Path": "/opt/conda/bin/flake8",
-		"python.linting.pycodestylePath": "/opt/conda/bin/pycodestyle",
-		"python.linting.pydocstylePath": "/opt/conda/bin/pydocstyle",
-		"python.linting.pylintPath": "/opt/conda/bin/pylint"
-	},
+    // Set *default* container specific settings.json values on container create.
+    "settings": {
+        "terminal.integrated.shell.linux": "/bin/bash",
+        "python.pythonPath": "/opt/conda/bin/python",
+        "python.linting.pylintEnabled": false,
+        "python.linting.enabled": false,
+        "python.linting.mypyEnabled": false,
+        "python.linting.pycodestyleEnabled": true,
+        "python.dataScience.alwaysTrustNotebooks": true,
+    },
 
-	// Add the IDs of extensions you want installed when the container is created.
-	"extensions": [
-		"ms-python.python"
-	]
+    // Add the IDs of extensions you want installed when the container is created.
+    "extensions": [
+        "ms-python.python"
+    ],
 
-	// Use 'forwardPorts' to make a list of ports inside the container available locally.
-	// "forwardPorts": [],
-
-	// Use 'postCreateCommand' to run commands after the container is created.
-	// "postCreateCommand": "python --version",
-
-	// Uncomment to connect as a non-root user. See https://aka.ms/vscode-remote/containers/non-root.
-	// "remoteUser": "vscode"
-}
+    // Use 'postCreateCommand' to run commands after the container is created.
+    "postCreateCommand": "sudo /opt/conda/bin/conda install --name base seaborn matplotlib -y",
```

Comment:
These packages were already part of the original image. Can't we have them on this new one instead of installing them manually as a post-create command?

The issue here is that post-create commands run after the codespace is already active (usable), and if the user executes `hello.py` before this command finishes executing, it will fail.